dealiiadapter.cc

```diff
@@ -37,7 +37,7 @@
 #include <deal.II/fe/fe_q.h>
 #include <deal.II/fe/mapping_q_eulerian.h>
 
-//#include "precice/SolverInterface.hpp"
+#include "precice/SolverInterface.hpp"
```

Comment:
Do you really need the "precice" prefix here?

Reply:
@uekerman scoped includes are good practice and a must if we aim to provide packages.

Reply:
This depends on the specification in `INCLUDE_DIRECTORIES` in the `CMakeLists.txt` file. I can also add the prefix there and skip it in the main file, if you prefer.

Reply:
:hand: I vote for the future-proof version with the prefix.

Reply:
Frederic is the expert here, so OK for me